### ISSUE TITLE
Format checking/fixing script breaks with python3.8

### ISSUE THREAD

**Issue by LisaLudique** [envoyproxy/envoy#12740]
Description:

Running `tools/code_format/check_format.py check` (or with `fix`) with python3.8.0 or python3.8.7 on MacOS fails with the following error:

```
➜  envoy git:(master) tools/code_format/check_format.py check
multiprocessing.pool.RemoteTraceback: 
"""
Traceback (most recent call last):
  File "/Users/lisalu/.pyenv/versions/3.8.0/lib/python3.8/multiprocessing/pool.py", line 125, in worker
    result = (True, func(*args, **kwds))
  File "/Users/lisalu/src/envoy/tools/code_format/check_format.py", line 965, in checkApiShadowStarlarkFiles
    if operation_type == "check":
NameError: name 'operation_type' is not defined
"""

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "tools/code_format/check_format.py", line 1152, in <module>
    error_messages += sum((r.get() for r in results), [])
  File "tools/code_format/check_format.py", line 1152, in <genexpr>
    error_messages += sum((r.get() for r in results), [])
  File "/Users/lisalu/.pyenv/versions/3.8.0/lib/python3.8/multiprocessing/pool.py", line 768, in get
    raise self._value
NameError: name 'operation_type' is not defined
```

While the argument is correctly parsed into the `operation_type` variable in the main function it is undefined/not visible to the workers running checkFormat. I am guessing this behavior changed with Python3.8 (https://bugs.python.org/issue39931?) as I am able to run the script successfully with Python3.7.8 and older versions.

**Comment by LisaLudique** [comment]
This applies to some of the other command line arguments like `build_fixer_check_excluded_paths`; I am working on a PR to pass those variables into the helper functions that use them.

**Comment by jingwei99** [comment]
I am running the same issue, and the fix from #12743 works for me, it'll be great for it to have it merged 🙌
